lib/internal/source_map/source_map_cache.js

```diff
@@ -73,6 +76,15 @@ function dataFromUrl(basePath, sourceMappingURL) {
   }
 }
 
+// Cache the length of each line in the file that a source map was extracted
+// from. This allows translation from byte offset V8 coverage reports,
+// to line/column offset Source Map V3.
+function lineLengths(content) {
+  return content.split('\n').map((line) => {
```

Comment:
is `content` guaranteed to only have lines separated by LF?

Reply:
@targos my thinking was that if the delimiter is `\r\n`, we take the `\r` into account as an additional character in the line length, if we split on `\n`. Then upstream tooling need only `.join('\n')`, and we'll have the appropriate line lengths for a Windows system.

Alternatively, we could split on `\r\n`, but upstream tooling needs to know to join on the appropriate delimiter, which seems a little annoying.

Reply:
Okay for `\r\n`. But what about `\r` alone, or other line terminators?

![image](https://user-images.githubusercontent.com/2352663/66418141-bc81c980-ea01-11e9-8f2f-036cd0192f47.png)

Reply:
open to suggestions, when writing tooling I've tended to only support the dominant `\r` and `\r\n` use-case, did you have a regex in mind?

Reply:
not necessarily, but if this is a known limitation, it should be specified in the documentation

Reply:
@targos rather than adding more documentation, why don't we split on `require('os').EOL` (I didn't realize this constant existed). And then it's the onus of the upstream tooling to `.join` on the same constant.

Reply:
that would change how the same source map is interpreted on different oses.

Reply:
@devsnek perhaps this slight modification:

```
/\n|\u2028|\u2029/
```

At which point the the `\r` character will be baked into the line length, and we can always simply join on `\n`, and I believe we'll end up with a correct report -- the offsets in the coverage will include the `\r`.